src/libcore/ptr/mod.rs

```diff
@@ -668,7 +668,18 @@ pub unsafe fn read<T>(src: *const T) -> T {
 /// ```
 ///
 /// Accessing unaligned fields directly with e.g. `packed.unaligned` is safe however.
-// FIXME: Update docs based on outcome of RFC #2582 and friends.
```

Comment:
This should remain, it refers to the statements above re: creating unaligned references and casting them to pointers immediately.